cmd/base/testdata/Onboard/sampleOnboardConfigWithCustomQuota.yaml

```diff
@@ -0,0 +1,19 @@
+env: MOC
```

Comment:
question: why do you have an extra directory `base` here? Why not put it all in `testdata` like in the `api` folder?

Reply:
I think previously `base` was the "root test directory" for cmd because there were no other overlay or component files that existed so everything could fit under base. I planned to change this but apparently never got back to it. Addressing in the next commit.